pkgs/desktops/gnome/extensions/no-title-bar/default.nix

```diff
@@ -23,7 +23,10 @@ stdenv.mkDerivation rec {
 
   makeFlags = [ "INSTALLBASE=$(out)/share/gnome-shell/extensions" ];
 
-  uuid = "no-title-bar@jonaspoehler.de";
+  passthru = {
+    extensionUuid = "no-title-bar@jonaspoehler.de";
+    extensionPortalSlug = "no-title-bar-forked";
```

Comment:
For consistency with other manual extensions that do not match, I would just drop this line.